datashadow/Shadow.java

```diff
@@ -0,0 +1,14 @@
+class Shadow
+{
+	int x=10;
+	void show(int x)
+	{
+	System.out.println("this is local variable :" + x);
```

Comment:
First letter of message should be capital

Reply:
this okay